### DISCUSSION TITLE
2023.11.17 第十三讲 认识年轻人 了解新世界 惭愧的课堂笔记

### DISCUSSION THREAD

**Discussion by guttasingapore** [RoadSideUniversity/ClubHouse2024#36] [General]
认识年轻人 了解新世界 clubhouse live  https://www.clubhouse.com/invite/sqwu7DT7

CBDC
https://www.mas.gov.sg/news/media-releases/2023/mas-lays-foundation-for-safe-and-innovative-use-of-digital-money-in-singapore
https://www.mas.gov.sg/development
https://www.coindesk.com/policy/2023/11/16/singapore-central-bank-to-start-live-wholesale-cbdc-trials/

talent
https://www.fintechfestival.sg/talent-zone
https://talentpavilion.sg/

blockchain in little town
https://singaporeblockchain.org/
https://www.metacamp.so/

**Comment by guttasingapore** [comment]
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/135027355/eb411994-3613-4b98-9411-6e47bac1fd25)

**Reply by Lewisrxliu** [reply]
背后原来有这个
https://solana.com/
https://www.reddit.com/r/solana/comments/xnkvq9/metacamp_solana_foundation_are_setting_up_the/?rdt=36023

**Reply by Lewisrxliu** [reply]
背后还有这个
https://www.skillsfuture.gov.sg/skillsreport

**Reply by Lewisrxliu** [reply]
> 背后原来有这个
> 
> https://solana.com/
> 
> https://www.reddit.com/r/solana/comments/xnkvq9/metacamp_solana_foundation_are_setting_up_the/?rdt=36023

Solana airdrop! ??? 
https://twitter.com/SolanaFloor/status/1742196996255998348?t=81bKeWPlmqEc6vwxniitAA&s=19

**Comment by Lewisrxliu** [comment]
金融行业面对百年未有之大变局。XP老师的问题，如果你是传统银行，现在应该怎么做，我的回答供大家讨论批评：拥抱新科技，重组资产，减负减重，参与制定新标准 重新定义风险管理体系 构建新杠杆 建立公开透明监管的信任机制。XP老师点评： 大道理没错，现在的问题是缺乏路径

**Comment by Lewisrxliu** [comment]
今天有位年轻人给我看了这个报道
https://www.asiaone.com/singapore/mother-2-loses-320k-life-savings-scam-everything-fell-apart

**Reply by ergatea** [reply]
骗子是没有国界的

**Comment by guttasingapore** [comment]
深感羞愧地纠正自己的一个错觉， 延续FFS现场体验，根据恒Sir指导寻找钱包未果的后记， 事实上，因为我的无知，险些错过了本地一场跨境钱包LiquidGroup在两个央行指导下的静悄悄表演, 我第一次听说LiquidPay, 也许是本地第一次paynow跨境，原来巨大的变革已经在眼皮底下发生了很多年，它们横空出世，起点可以与普通人无关，最终不可逆，终点与所有人大相关。
https://www.zaobao.com.sg/realtime/singapore/story20231117-1450640?amp
https://liquidgroup.sg/liquidpay-app-paynow/
https://www.zaobao.com.sg/zfinance/realtime/story20210208-1122712
https://github.com/orgs/RoadSideUniversity/discussions/33

**Reply by guttasingapore** [reply]
缩写索引
VPA - Virtual Payment Address (VPA), 
UPI - Unified Payments Interface (UPI)
NFI - Non-Bank Financial Institutions (NFIs)
MPM - merchant-presented-mode (MPM) 
CPM - customer-presented-mode (CPM)
https://xnap.network/  
Operated by Liquid Group, XNAP is Asia’s fastest growing digital cross-border QR payment network. The network provides a smart and secure market infrastructure, supporting cross-border C2B Payments of national schemes, e-wallets and card-based digital wallets.

**Comment by guttasingapore** [comment]
继续深感羞愧地纠正自己的一大误区，原来破县已经存在超过16种电子钱包， 其中Nets,EZ-Link几十年本地化如同空气无处不在， paynow/paylah过去5年几乎是千家万户的必备，鄙人使用多年而不自知，羞愧啊羞愧。原来一切已经在过程中，未来永远在变化中
https://www.scs.org.sg/articles/e-wallet-singapore

**Reply by Lewisrxliu** [reply]
我的惭愧居然超越了国界，20多年一挥间，曾经的TNG继续领导自驾刚需，竞争不息垄断不止，近两年在阿里系的支撑下继续发展，脑补在马的各种支付钱包已经进化成什么样的格局了，未来每个商家每个人都必须有无数个钱包吗？
https://www.touchngo.com.my/consumer/toll/card/

**Reply by Lewisrxliu** [reply]
惭愧，我的小小尝试，发现到今天为止最后一步还是由卡司来帮助我完成的，背后构建其实是阿里，传统银行跨界跨境跨平台好难哦
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/e5eee473-7f37-4ba8-b23e-a6cd3ff871da)

**Comment by ergatea** [comment]
workspace 非常不错，规模很大

lewis ***@***.***>于2023年11月20日 周一17:39写道：

**Reply by guttasingapore** [reply]
还有这个workspace待探索
https://www.scs.org.sg/resources

**Comment by ergatea** [comment]
不要参与这个空投

lewis ***@***.***>于2024年1月3日 周三12:41写道：

**Reply by Lewisrxliu** [reply]
感谢提醒。我不会参加这个空投。
